### PR TITLE
fix: throw correct error message for DOW range when dayOfWeekStartIndexZero is false

### PR DESCRIPTION
**Pull request by OwenVey**

### CODE REVIEW

**Comment by OwenVey** [comment]
refers to issue #201

File: src/rangeValidator.ts [hunk 1]
```diff
@@ -55,12 +55,15 @@ export default class RangeValidator {
...
   }
 
-  static dayOfWeekRange(parse: string) {
+  static dayOfWeekRange(parse: string, dayOfWeekStartIndexZero: boolean) {
     const parsed = parse.split(',');
     for (let i = 0; i < parsed.length; i++) {
       if (!isNaN(parseInt(parsed[i], 10))) {
         const dayOfWeek = parseInt(parsed[i], 10);
-        assert(dayOfWeek >= 0 && dayOfWeek <= 6, 'DOW part must be >= 0 and <= 6');
+        assert(
```
Comment (bradymholt) [inline src/rangeValidator.ts:63]: Won't this validation trigger when `7` is supplied, even when `! dayOfWeekStartIndexZero`?

Comment (OwenVey) [reply src/rangeValidator.ts:63]: Inside of the ```normalize()``` function in ```cronParser.ts```, the dow is shifted down 1 when ```dowDigitsAdjusted``` is false. This means that when 7 is supplied, it is shifted down to 6 before being passed to the validator. However, the error message currently is saying that the DOW must be less than or equal to 6. So basically just shifting the range of allowed numbers up 1 in the error message when ```dowDigitsAdjusted``` is set to false.

Comment (OwenVey) [reply src/rangeValidator.ts:63]: Just to clarify, the rangeValidator is working correctly, even when ```dowDigitsAdjusted``` is false, it's just that the error message is incorrect whenever ```dowDigitsAdjusted = false``` and the user supplies a number less than 1 or greater than 7.

Comment (bradymholt) [reply src/rangeValidator.ts:63]: 👍

**Comment by bradymholt** [comment]
Thanks!

**Pull request merged**